content/en/docs/actionners/list.md

```diff
@@ -197,4 +197,27 @@ Example:
   actionner: calico:networkpolicy
   parameters:
     order: 20
+```
+
+### `aws:lambda`
+
+* Description: **Invoke an AWS lambda forwarding the output_fields**
+* Continue: `true`
+* Parameters:
+  * `aws_lambda_name`: Lambda name to call. Lambda must reside in the same region as your default credential provider or static region provided in configuration.
+  * `aws_lambda_alias_or_version`: Lambda alias or version to call. (default: $LATEST)
+  * `aws_lambda_invocation_type`: Invocation type for Lambda. Accepted values: RequestResponse, Event, DryRun. (default: RequestResponse)
+* Required AWS access:
+  * `sts:getCallerIdentity`
+  * `lambda:InvokeFunction`
+* Source: `cloudtrail`
```

Comment:
technically, it could work with any source. on the user to develop the action

Reply:
Yes, I wasn't 100% sure what to specify here, how we can we say all of them? any?

Reply:
`any` is find